### PR TITLE
Add @devel and @demos teams to codeowners for examples

### PR DESCRIPTION
**Pull request by nuke-web3**
https://github.com/orgs/risc0/teams - looking there this seems like the best fit for at least one of us to approve, mostly this is to ensure that those teams are notified of changes as examples are mission critical to @risc0/devrel and likely impact or even suggest breaking changes to @risc0/demos .

Ont thing though - we should not block merges - so maybe we want to add `@risc0/platform` or even `@risc0/dev` to ensure changes can be made without waiting on us.

Closes https://github.com/risc0/devrel/issues/99 .

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -9,6 +9,8 @@
 /rzup/                                 @risc0/platform @hmrtn
 /web/                                  @risc0/platform @nahoc
 /website/                              @risc0/devrel @pdg744
+/examples/                             @risc0/devrel @risc0/demos
+
```
Comment (nuke-web3) [inline .github/CODEOWNERS:13]:
```suggestion
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -9,6 +9,7 @@
 /rzup/                                 @risc0/platform @hmrtn
 /web/                                  @risc0/platform @nahoc
 /website/                              @risc0/devrel @pdg744
+/examples/                             @risc0/devrel @risc0/demos
```
Comment (flaub) [inline .github/CODEOWNERS:12]:
```suggestion
/examples/                             @risc0/devrel @risc0/demos @risc0/platform
```

Also could we sort the dirs?

**Review by flaub — APPROVED** [review]

File: .github/CODEOWNERS (hunk 2 above)
Comment (nuke-web3) [reply .github/CODEOWNERS:11]: added comments to sections and did a sort ascending. LMK if that is what you wanted?

Comment (nuke-web3) [reply .github/CODEOWNERS:11]: https://github.com/risc0/risc0/blob/86dd3dba48d601316a239f97fae34b56162af1c4/.github/CODEOWNERS

Comment (nuke-web3) [reply .github/CODEOWNERS:11]: Note that there is a global requirement on platform team already :
image

**Review by flaub — APPROVED** [review]

**Pull request merged**